girder/girder_large_image/web_client/vue/components/FrameSelector.vue

```diff
@@ -135,6 +137,8 @@ export default Vue.extend({
 
 <template>
     <div class="image-frame-control-box">
+        <div id="frame_id" style="display: none;">{{ currentFrame }}</div>
```

Comment:
I'd like to propose different element IDs for these: `current_image_frame` and `current_image_style`.